src/app/pages/charging-stations/charging-station/charging-station.component.ts

```diff
@@ -97,8 +97,12 @@ export class ChargingStationComponent implements OnInit {
   public saveChargingStation(chargingStation: ChargingStation) {
     // Clone
     const chargingStationToSave = Utils.cloneObject(chargingStation) as ChargingStation;
-    // Do not save charge point
-    delete chargingStationToSave.chargePoints;
+    if (!chargingStationToSave.manualConfiguration) {
+      // Do not save charge point
+      delete chargingStationToSave.chargePoints;
+    } else {
+      this.adjustChargePoints(chargingStationToSave);
+    }
```

Comment:
We need it because if the user modifies e.g. amperage or number of phases of the connectors, the charge point will not have the right power values anymore, which leads to inconsistencies.

So e.g. we have a three phased station with a three phased charge point and a three phased connector.  
If the user sets the connector to one phase, the charge point does not have the right number of phases and power anymore, which would cause trouble e.g. in static limitation or charging profiles.